Quaver.Shared/Screens/Select/UI/Modifiers/DrawableModifierBool.cs

```diff
@@ -31,7 +32,15 @@ public override List<DrawableModifierOption> CreateModsDialogOptions()
                 new DrawableModifierOption(this, "ON", (o, e) =>
                 {
                     if (!ModManager.Mods.HasFlag(Modifier.ModIdentifier))
+                    {
                         ModManager.AddMod(Modifier.ModIdentifier);
+                        if (Modifier.Name == "No Slider Velocities"
```

Comment:
String comparison isn't a good way to go about it. What if we wanted to change the name of the modifier?

[IGameplayModifier](https://github.com/Quaver/Quaver/blob/develop/Quaver.Shared/Modifiers/IGameplayModifier.cs) has a property that defines if the mod is ranked or not. 

The warning notification *could* go in [ModManager.AddMod()](https://github.com/Quaver/Quaver/blob/develop/Quaver.Shared/Modifiers/ModManager.cs#L54) if you wanted to approach it this way and check against the created instance of the modifier if it's ranked or not.

However, I think that having some sort of indicator in the mod menu itself would be better than a notification.